website/docusaurus.config.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
       src: 'https://widget.surveymonkey.com/collect/website/js/tRaiETqnLgj758hTBazgd8ryO5qrZo8Exadq9qmt1wtm4_2FdZGEAKHDFEt_2BBlwwM4.js',
       defer: true,
     },
-    {src: 'https://snack.expo.dev/embed.js', defer: true},
+    {src: 'https://snack.expo.dev/embed.js?version=20230517', defer: true},
```

Comment:
You can remove it, but I noticed a bug on our end with caching (why didn't anyone tell me caching is hard 🤷). Either way, for now, this works and soon it will work without too 😄

Reply:
A day has passed, and our cached embed is updated! Meaning:

```suggestion
    {src: 'https://snack.expo.dev/embed.js', defer: true},
```